### PR TITLE
Documentation API by swagger

### PR DESCRIPTION
**Pull request by tchevily**
# Description

This PR adds documentation of API using swagger format

## Issue (optional)

Issue link: BOT-245

## Pull Request type (optional)

This is a new feature

## How to test

Here are the following steps to test this pull request:

- load content of documentation/swagger.yml file inside swagger editor

### CODE REVIEW

File: documentation/swagger.yml [hunk 1]
```diff
@@ -0,0 +1,2060 @@
...
+                description: version of db migration
+              navitia_url:
+                type: string
+                description: URL of navitia Web Service
+              rabbitmq_info:
+                type: object
+                description: Information about configuration of RabbitMQ
+                properties:
+                  alternates:
+                    type: array
```
Comment (dvdn) [inline documentation/swagger.yml]: Semantic error at paths./status.get.responses.200.schema.properties.rabbitmq_info.properties.alternates
Schema objects with 'array' type require an 'items' property

File: documentation/swagger.yml [hunk 2]
```diff
@@ -0,0 +1,2060 @@
...
+################################################################################
+info:
+  version: v1
+  title: Documentation of API Chaos
+  description: |
+    Chaos is the web service which can feed [Navitia](https://github.com/CanalTP/navitia) with real-time [disruptions](http://doc.navitia.io/#traffic-reports).
+
+    It can work together with [Kirin](https://github.com/CanalTP/kirin) which can feed [Navitia](https://github.com/CanalTP/navitia) with real-time delays.
+
+    Chaos manage disruptions and help you to communicate with your travellers on the best.
```
Comment (dvdn) [inline documentation/swagger.yml]: travelers

File: documentation/swagger.yml [hunk 3]
```diff
@@ -0,0 +1,2060 @@
...
+     * a Navitia token, allowing you to request navitia on a data coverage
+     * a customer ID
+     * a contributor ID
+
+    Your usual Kisio Digital interlocutor can provide you these elements, and an access to the production or pre-production platform.
+    Before using the API in production, you will need to provide:
+     * integration specifications
+     * expected load your application will generate.
+
+    These two points are recquired to help us managing the Chaos platform.
```
Comment (dvdn) [inline documentation/swagger.yml]: required // manage Chaos platforms

**Comment by dvdn** [comment]
You can also add the link in API in readme :
http://petstore.swagger.io/?url=https://raw.githubusercontent.com/CanalTP/Chaos/master/documentation/swagger.yml

File: documentation/swagger.yml (hunk 1 above)
Comment (tchevily) [reply documentation/swagger.yml]: Done

**Comment by dvdn** [comment]
Why a select box with only a unique choice ? https://github.com/CanalTP/Chaos/blob/b4acacd508080e902668df1a3a5ccdf05730e117/documentation/swagger.yml#L40


Are we sure Swagger doesn't track users inputs using documentation and dynamic calls as somes are private (i.e. Navitia customer private Tokens) ?

**Pull request merged**